recipes/llvmlite/meta.yaml

```diff
@@ -21,9 +21,11 @@ requirements:
     - python
     - setuptools
     - enum34  # [py2k]
+    - llvmdev
   run:
     - python
     - enum34  # [py2k]
+    - 
```

Comment:
This is causing a crash, because it's trying to install the package with an empty name, and no such package exists. :)

Reply:
My bad. I must have done it by mistake.